### ISSUE TITLE
Relayer dupe checking & legitimate dupes

### ISSUE THREAD

**Issue by mikeshultz** [OriginProtocol/origin#2783]
Need to do a review and consider when there may be legitimate duplicate requests.  For instance, if the risk engine rejects a request, would the user potentially have the opportunity to send it again later? 

Most should be fine, for instance, if there's a pending tx rejection or a seen nonce error, we never want to see that again.  But there might be some circumstances not taken into account.

@franckc @nick Thoughts?

**Comment by franckc** [comment]
Yes, if the risk engine rejects a tx, the user would be potentially be able to resubmit later and have it accepted. For example if the rejection was due to rate limiting. But I would expect this to be a very rare corner case.
